### PR TITLE
apps sc: fix WIP-CHANGELOG.md

### PR DESCRIPTION
**Pull request by robinelastisys**
**What this PR does / why we need it**:
I messed up and managed to merge the old WIP-CHANGELOG into the new one in my latest merge. This needs to be fixed.

**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [ ] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

**Review by robinAwallace — APPROVED** [review]

File: WIP-CHANGELOG.md [hunk 1]
```diff
@@ -4,40 +4,8 @@
...
-- Prometheus alert and servicemonitor was separated
-- Default user alertmanager namespace changed from monitoring to alertmanager.
-- Reworked configuration handling to keep a read-only default with specifics for the environment and a seperate editable override config for main configuration.
-- Integrated secrets generation script into `ck8s init` which will by default generate password and hashes when creating a new `secrets.yaml`, and can be forced to generate new ones with the flag `--generate-new-secrets`.
 - The falco grafana dashboard now shows the misbehaving pod and instance for traceability
+- Reworked configuration handling to use a common config in addition to the service and workload configs. This is handled in the same way as the sc and wc configs, meaning it is split between a default and an override config. Running `init` will update this configuration structure, update and regenerate any missing configs, as well as merge common options from sc and wc overrides into the common override.
```
Comment (viktor-f) [inline WIP-CHANGELOG.md:8]: This should not be part of this PR.
```suggestion
```

**Review by viktor-f — CHANGES_REQUESTED** [review]

File: WIP-CHANGELOG.md (hunk 1 above)
Comment (robinelastisys) [reply WIP-CHANGELOG.md]: Done!

**Review by aarnq — APPROVED** [review]

**Review by viktor-f — APPROVED** [review]
